web/src/message_edit.js

```diff
@@ -578,10 +579,10 @@ function start_edit_with_content($row, content, edit_box_open_callback) {
     if (edit_box_open_callback) {
         edit_box_open_callback();
     }
-
-    upload.setup_upload({
+    const row = rows.id($row);
```

Comment:
Name this `row_id` to be clear it's an id and not an object.

Reply:
Yes, you're right, thanks! Naming variables can sometimes be the trickiest part of programming, right?